src/Elf.h

```diff
@@ -269,6 +269,13 @@ class Section {
     }
     uint64_t get_alignment() const { return alignment; }
 
+    void fixup_relocations_offset(uint64_t offset_fixup) {
```

Comment:
"fixup" is a bit ambiguous. I think this function should just be inlined into it's only user (prepend_contents) below. The loop body is reasonably a one-liner:

    ri->set_offset(ri->get_offset() + offset_fixup);

(If I didn't define operator -> on these iterators... we should.)

Reply:
Oh, also, a range based for loop should work:

    for (Relocation& r : sec.relocations()) {
        r.set_offset(r.get_offset() + offset);
    }